styles/style.css

```diff
@@ -69,6 +69,11 @@ button:hover {
     padding: 16px 0px;
 }
 
+.left-menu nav a:focus {
+    background-color: var(--third-color);
+    outline: none;
+}
```

Comment:
Important feature to style selected elements when the user is not navigating with a mouse.

Reply:
Thanks for that information :D